<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>701</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="logolabel">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>60</y>
       <width>141</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
image: url(:/resources/img/logo.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="eLogin">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>300</y>
       <width>251</width>
       <height>43</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

background-color:#ffffff;
border:#18191e;
color:#000000;
padding-left:25px;
border-radius:10px;

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string extracomment="password">Account ID...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>363</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
color:#f9ce3b;
}</string>
     </property>
     <property name="text">
      <string>Remember Account ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rememberMeButton">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>358</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background-color:#191a1f;

border-image: url(:/resources/img/uncheck.png);
border:0px;

}

QPushButton:checked
{

border-image: url(:/resources/img/check (3).png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="signin">
     <property name="geometry">
      <rect>
       <x>444</x>
       <y>442</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background-color:#f8cd3b;
border-radius:20px;
color:#32353e;
padding-bottom:5px;
}

</string>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="upgrade">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>531</y>
       <width>131</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background-color:#191a1f;
border-radius:12px;
border-style:solid;
color:#c4c5c9;
border:1px solid #ffb400;
padding-left:5px;
padding-right:5px;

}</string>
     </property>
     <property name="text">
      <string>CREATE ACCOUNT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>530</y>
       <width>211</width>
       <height>37</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
color:#ffffff;
}</string>
     </property>
     <property name="text">
      <string>I  LOST  MY  ACCOUNT  NUMBER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>554</x>
       <y>451</y>
       <width>27</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
image: url(:/resources/img/signarrow.PNG);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
